coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -223,11 +440,82 @@ export default {
   computed: {
     ...mapState(['corpNum', 'agmDateValid']),
 
+    // today's date as YYYY-MM-DD, current timezone
+    today () {
+      let today = new Date()
+      today = '' +
+        today.getFullYear() + '-' +
+        (today.getMonth() * 1 + 1).toString().padStart(2, '0') + '-' +
+        today.getDate().toString().padStart(2, '0')
```

Comment:
Just an aside - I think we should eventually look into using some sort of date library, like Moment. We'll be doing a lot of date manipulation, comparison, etc. Something to ponder, for future work.

Reply:
Good point - I actually think that the AGMDate component might do this better, but I was doing this at midnight last night :) Will re-evaluate with the next batch of changes because I have more date stuff to add.